src/common/datamodeltypes/canrawsenderdata.h

```diff
@@ -0,0 +1,8 @@
+#ifndef CANRAWSENDERDATA_H
+#define CANRAWSENDERDATA_H
+
+#include "candevicedata.h"
+
+typedef CanDeviceDataIn CanRawSenderDataOut;
```

Comment:
Different data types on in and out ports will prevent creating loop in CanDevice.
As there is no detailed specification for each component I did not know if this should be allowed or not, so I decided to put it in PoC. This is an easy fix but will strongly affect nodes behavior.